lib/module.js

```diff
@@ -250,7 +250,7 @@ Module._resolveLookupPaths = function(request, parent) {
 
   // make sure require('./path') and require('path') get distinct ids, even
   // when called from the toplevel js file
-  if (parentIdPath === '.' && id.indexOf('/') === -1) {
+  if (parentIdPath === '.' && id.includes('/')) {
```

Comment:
i agree, @thefourtheye- will take a look and maybe send in another PR :) - not relevant to this one, but in another one- would make more sense yeah.